src/kernel.cpp

```diff
@@ -63,6 +64,7 @@ static std::map<int, unsigned int> mapStakeModifierTestnetCheckpoints =
     (336000, 0x8330dc09u )
     (372751, 0xafb94e2fu )
     (382019, 0x7f5cf5ebu )
+    (408500, 0x68cadee2u )
```

Comment:
are you sure this line is right?

Reply:
Yes. Check here: https://chainz.cryptoid.info/ppc-test/block.dws?408500